vllm/model_executor/layers/quantization/moe_quant_int.py

```diff
@@ -0,0 +1,362 @@
+from typing import Any, Callable, Dict, List, Optional
+
+import torch
+
+from vllm.distributed import get_tensor_model_parallel_rank, get_tp_group
+from vllm.model_executor.layers.fused_moe.layer import (
+    FusedMoE, FusedMoEMethodBase, FusedMoeWeightScaleSupported)
+from vllm.model_executor.layers.linear import (LinearBase,
+                                               UnquantizedLinearMethod)
+from vllm.model_executor.layers.quantization.awq_marlin import (
+    AWQMarlinConfig, AWQMarlinLinearMethod)
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig, QuantizeMethodBase)
+from vllm.model_executor.layers.quantization.gptq_marlin import (
+    GPTQMarlinConfig, GPTQMarlinLinearMethod)
+from vllm.model_executor.utils import set_weight_attrs
+
+
+class MoeQuantIntConfig(QuantizationConfig):
+    """Config class for Int8 experts quantization."""
```

Comment:
Update this comment

Is there any more specific name we could use for this method? I also feel that `--quantization moe_quant_int` is not clear. Maybe you could change to `--quantization moe_wNa16` and `MoeWNA16Config`? Open to other names

Reply:
moe_wNa16 is a better name, I would change it.